server/test/lib/device/device.newStateEvent.test.js

```diff
@@ -121,4 +121,13 @@ describe('Device.newStateEvent', () => {
     expect(newDeviceFeature).to.have.property('last_value_changed');
     expect(newDeviceFeature.last_value_changed).to.deep.equal(dateInTheFuture);
   });
+  it('should not save state missing device feature', async () => {
+    const stateManager = new StateManager(event);
+    const device = new Device(event, {}, stateManager, {}, {}, {}, job);
+    await device.newStateEvent({
+      device_feature_external_id: 'hue:binary:1',
+      state: 12,
+      created_at: '2019-02-12 07:49:07.556 +00:00',
+    });
```

Comment:
Can you check that the state was indeed not saved?

Reply:
Ok, I had a look, by the way the  catch block is hiding an issue due to DB not available during test (original: [Error: SQLITE_ERROR: no such table: t_device_feature]). I provide new check (I also change flow for newStringState)

Reply:
Ok weird, do you have this issue locally or on CI ? The table should be ready when the tests starts